### PR TITLE
backend image functionality and both frontend and backend stuff for multi line content

### PR DESCRIPTION
**Pull request by Chrisvann1**
I added a table to note.db called images that works basically the same as the tags table. I then created the add and delete functionality for this image table. This also has the multi line content things added to it for both the frontend and the backend. Allowing the user to type multiple lines and get multiple lines returned to them.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -41,7 +48,7 @@ def init_db():
 # make sure you have Content-Type: application/json in the request header
 # payload JSON should have the following fields:
 # 'title' (required): string of note title
-# 'content' (optional): string of note content
+# 'content' (optional): list of strings for content
```
Comment (augustus-thomas) [inline app.py:51]: why was this added?

File: app.py [hunk 2]
```diff
@@ -102,6 +109,31 @@ def create_tag():
...
+    payload = request.json
+    try: 
+        title = payload.get('title')
+        if title is None: 
+            raise KeyError("Data Entry Requires A Title")
+    except KeyError as error:
+        abort(206, error)
+
+    try: 
+        images = payload.get('image')
```
Comment (augustus-thomas) [inline app.py:125]: should we validate the image path exists here? i know this is a link to an image, but should it be backend responsibility to throw an error here?

File: frontend.py [hunk 3]
```diff
@@ -142,12 +142,19 @@ def runtime(state):
...
-					apiCalls.createNote(newNoteName,"")
-				else:
-					apiCalls.createNote(newNoteName,newContent)
+				printColor("What content would you like to add to start? (If you wish to add content later, enter '*END'.)", getConfig(2),"")
+				printColor("*END: Stop typing to file.",getConfig(2),"")
+				printColor("return key: start a new line.",getConfig(2),"\n")
+				newContent1 = []
+				while True: 
+					notes = input("")
+					if notes == "*END":
```
Comment (augustus-thomas) [inline frontend.py:151]: looks like we are going line by line? so to exit you have to type *END on its own line. shouldn't we say that it needs to be its own line in the user instructions above?

**Review by augustus-thomas — CHANGES_REQUESTED** [review]
Just two things you should address before I merge:
1. Should the backend validate that the image path provided indeed exists?
2. Should we tell the user that *END needs to be on its own line?

Don't really care what your responses are, it's fine if you say No to both lol. Just want a response before I approve.

File: frontend.py (hunk 3 above)
Comment (Chrisvann1) [reply frontend.py:151]: I think it would be a good idea to specify that it needs to be on its own line. I can do that later today.

File: app.py (hunk 2 above)
Comment (Chrisvann1) [reply app.py:125]: I believe it should be both on the frontend and the backend to do check for this. The backend is saving the image path in the database to connect it to a note so I think the backend should check for the image in this regard. I believe the front in should check if the image exists locally in a directory on the user's computer.

File: app.py (hunk 1 above)
Comment (Chrisvann1) [reply app.py:51]: This was a mistake lol. I can change this

**Review by augustus-thomas — APPROVED** [review]
We are NOT validating image path, which is fine by me!
